presto-main/src/main/java/com/facebook/presto/cost/PlanNodeStatsEstimateMath.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.cost;
+
+import java.util.stream.Stream;
+
+public class PlanNodeStatsEstimateMath
+{
+    private PlanNodeStatsEstimateMath()
+    {}
+
+    @FunctionalInterface
+    private interface RangeSubtractionStrategy
+    {
+        StatisticRange range(StatisticRange leftRange, StatisticRange rightRange);
+    }
+
+    public static PlanNodeStatsEstimate differenceInStats(PlanNodeStatsEstimate left, PlanNodeStatsEstimate right)
+    {
+        return differenceInStatsWithRangeStrategy(left, right, StatisticRange::subtract);
```

Comment:
remove suffix `WithRangeStrategy`

Reply:
Note: the 3rd argument is `RangeSubtractionStrategy`, not `SymbolStatsEstimateSubtractionStrategy`.